src/components/TabList/index.module.scss

```diff
@@ -0,0 +1,145 @@
+@import "../styles/foundation.scss";
+
+$size: 32px;
+
+.container {
+  width: 100%;
+}
+
+.tabList {
+  width: inherit;
+
+  display: flex;
+
+  margin: unset;
+  padding: unset;
+
+  transform: translateY(4px);
+
+  list-style: none;
+
+  &__tab {
+    display: flex;
+
+    justify-content: center;
+    align-items: center;
+
+    cursor: pointer;
+
+    span {
+      width: 100%;
+
+      padding: 0 12px;
+
+      text-align: center;
+
+      display: inline-block;
+
+      overflow: hidden;
+      white-space: nowrap !important;
+      text-overflow: ellipsis;
+    }
+
+    &:not(.tabList__tab__selected, .tabList__tab__selectedFirst) {
+      width: 148px;
+      min-width: 148px;
+
+      height: 40px;
+
+      border-width: $defaultBorder $defaultBorder 0 $defaultBorder;
+      border-style: solid;
+      border-radius: $borderRadius $borderRadius 0 0;
+
+      span {
+        color: $neutral700;
+      }
+    }
+
+    &:not(:first-of-type, .tabList__tab__selected) {
+      margin-left: -1px;
+    }
+
+    &__selected,
+    &__selectedFirst {
+      height: 44px;
+
+      position: relative;
+
+      transform: translateY(-2px);
+      -webkit-transform-style: preserve-3d;
+
+      &::before,
+      &::after {
+        content: "";
+
+        height: inherit;
+        width: inherit;
+
+        position: absolute;
+
+        cursor: pointer;
+      }
+
+      &::after {
+        top: 1px;
+      }
+
+      span {
+        align-self: center;
+        transform: translateZ(1px);
+
+        color: $neutral800;
+      }
+    }
+
+    &__selectedFirst {
+      width: 153px;
+      min-width: 153px;
```

Comment:
Why two widths?

Reply:
Because width and min-width have different options